Makefile

```diff
@@ -0,0 +1,28 @@
+IMAGE_BUILD_CMD = $(shell which podman 2>/dev/null || which docker)
+IMAGE_REGISTRY ?= "quay.io"
+REGISTRY_NAMESPACE ?= "modh"
+IMAGE_NAME="opendatahub-tests"
+IMAGE_TAG ?= "latest"
+
+FULL_OPERATOR_IMAGE ?= "$(IMAGE_REGISTRY)/$(REGISTRY_NAMESPACE)/$(IMAGE_NAME):$(IMAGE_TAG)"
+POETRY_BIN = poetry
```

Comment:
I think this is not needed if we switch to uv?

Reply:
leftover, removing